types/v56/types.go

```diff
@@ -397,7 +397,7 @@ type OrgVDCNetwork struct {
 // Description: Contains a list of VMware virtual hardware versions supported in this vDC.
 // Since: 1.5
 type SupportedHardwareVersions struct {
-	SupportedHardwareVersion []string `xml:"SupportedHardwareVersion,omitempty"` // A virtual hardware version supported in this vDC.
+	SupportedHardwareVersion []Reference `xml:"SupportedHardwareVersion,omitempty"` // A virtual hardware version supported in this vDC.
```

Comment:
Strangely enough, the XML unmarshaling was getting the right string into the result (probably because this type is an extension of a previous type, which is a string). But I prefer to have the right data structure, in case it should be used elsewhere.